src/backend/handler/gamestatus/gamestatushandler.ts

```diff
@@ -0,0 +1,100 @@
+import { GameStatus } from './../../../common/types'
+import { getMainWindow } from '../../utils'
+import Store from 'electron-store'
+
+const gameStatusHandlerStore = new Store({
+  cwd: 'store',
+  name: 'progress-handler'
+})
+
+function initGameStatusHandler() {
+  const gameStatusHandlers = getAllGameStatus()
```

Comment:
this is just me being nit picky, but maybe we can rename this variable and the `gameStatusHandlerStore` as `handlers` and `store`?

we have `gameStatus...` in almost every line and it makes it look like it's all the same